paper-toast.html

```diff
@@ -193,6 +197,26 @@
         },
 
         /**
+         * Called on transitions of the toast, indicating a finished animation
+         *
+         * @private
+         */
+        __onTransitionEnd: function(e) {
+          // there are at least three different transitions that are happening when opening and
+          // closing the toast. The last one so far is for `opacity`.
+          // This marks the end of the transition, so we check for this to determine if this
+          // is the correct event.
+          var isCorrectEvent =
```

Comment:
instead of creating a variable `isCorrectEvent` and checking for it twice, what about a big `if` with this check?
```javascript
if (e && e.target === this && e.propertyName === 'opacity') {
  if (this.opened) {
  //...